NuRadioReco/modules/channelLengthAdjuster.py

```diff
@@ -11,6 +11,8 @@ class channelLengthAdjuster:
     def __init__(self):
         self.number_of_samples = None
         self.offset = None
+        logger.warning("In most cases it is advisable to run a trigger module and use the channelReadoutWindowCutter module to cut the traces to the readout window \
+                       instead of this simple module.")
```

Comment:
Formatting (don't use `\`, just close and open the string, otherwise you get a bunch of whitespace)